app/models/cart_item.rb

```diff
@@ -1,4 +1,17 @@
 class CartItem < ActiveRecord::Base
   belongs_to :cart
   belongs_to :pricing
+  has_one :product, through: :pricing
+
+  def product_name
+    product.try(:name) || ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.